clients/client-core/src/client/real_messages_control/acknowledgement_control/retransmission_request_listener.rs

```diff
@@ -113,10 +117,10 @@ where
 
         // send to `OutQueueControl` to eventually send to the mix network
         self.real_message_sender
-            .unbounded_send(vec![RealMessage::new(
-                prepared_fragment.mix_packet,
-                frag_id,
-            )])
+            .unbounded_send((
+                vec![RealMessage::new(prepared_fragment.mix_packet, frag_id)],
```

Comment:
Looking at how those are being sent in channels in multiple places, would it perhaps make sense to make `TransmissionLane` part of the `RealMessage` struct instead of sending tuples everywhere?

Reply:
Well I just tried making this change, yes and no :)
At first it seems to make things more tidy. But the lane is per chunk of messages, so keeping it separate makes it easy to add a single burst of traffic to the relevant buffer in the `OutQueueControl`